ICAFIX/ApplyAutoReclean.sh

```diff
@@ -0,0 +1,263 @@
+#!/bin/bash
```

Comment:
The name of this script is extremely close to the `Pipeline` one, which will result in confusion.  Please explain why both of these scripts exist (with a focus on how they are different), so we can decide whether to rename at least one of them.

Reply:
`ApplyAutoReclean` runs the following steps:
- compute additional features
- inference by python models
- generate txt files with reclassified components

`ApplyAutoRecleanPipeline` does the following three steps:
- run `ApplyAutoReclean`
- apply handclassification
- run the actual cleanup stage

Reply:
If `ApplyAutoReclean` is not intended to be run directly by the user, it should be in a `scripts/` subdirectory.  I would still argue it should be renamed, perhaps `RecleanClassify.sh`.